Improv bluetooth sample (#223)
nanoframework/Samples: samples/Bluetooth/ImprovWifi/README.md

# Improv Wifi provisioning

This sample shows the use of the *Improv class* to provision the Wifi credentials for an ESP32 device via Bluetooth LE. 

The device will advertise with the name "Improv sample" and support the *Improv Service* allowing the Wifi Credentials to be setup directly from the Improv test page.
or any other web page.

This is an initial working version of the *Improv* class showing what can be done. 

> Probably later this class can be moved to a separate repo. so it has its own nu-get to make it easy to include in you own application.

Reviewer: Maybe something like this... 

```suggestion
Probably later this class can be moved to a separate repo, so it has its own NuGet package to make it easy to consume by applications.
```
